R/hms.R

```diff
@@ -274,3 +274,41 @@ print.hms <- function(x, ...) {
   cat(format(x), sep = "\n")
   invisible(x)
 }
+
+# Sequence generation -----------------
+
+#' @rdname hms
+#' @inheritParams base::seq
+#' @export
+seq.hms <- function(
+    from = hms(1),
+    to = hms(1),
+    by = NULL,
+    ...
+) {
+
+  if (!is_hms(to)) {
+    abort(sprintf(
+      "`to` isn't of class `hms` (current class: `%s`).",
+      class(to)[1]
+    ))
+  }
+  
+  from <- vec_cast(from, numeric())
+  to <- vec_cast(to, numeric())
```

Comment:
For symmetry and safety?

```suggestion
  from <- vec_cast(as_hms(from), numeric())
  to <- vec_cast(as_hms(to), numeric())
```

Reply:
Is this necessary? If from isn't hms, seq.hms shouldn't be called.